invadedhouse/models/room.py

```diff
@@ -23,7 +25,7 @@ def enter(self, player):
         self.look(player)
 
     def look(self, player):
-        print(self.get_name() + ':')
+        cprint(self.get_name() + ':', 'cyan')
```

Comment:
umm whats `cprint`?

Reply:
and whats termcolor

Reply:
it's part of a library that allows us to print IN COLOR

Reply:
WOAH!

Reply:
AWESOME

Reply:
termcolor is the library, cprint is a function in the library. you'll see a nice surprise when you implement the "monsters attack back" feature...

Reply:
btw IDLE always prints in blue. But probably anything you do with this printing in color thing wont be blue in IDLE. But we have IDE!

Reply:
also could we put optional instructions in the readme that will show people how to get IDE

Reply:
but PRINTING IN COLOR WOAH

Reply:
also what do i need to do to use this color thing because i need to install it i think right?